src/zepto.js

```diff
@@ -437,9 +437,10 @@ var Zepto = (function() {
 
     ready: function(callback){
       // don't use "interactive" on IE <= 10 (it can fired premature)
+      // pipe any ready callback call over task/event queue (#1235)
       if (document.readyState === "complete" ||
           (document.readyState !== "loading" && !document.documentElement.doScroll))
-        callback($)
+        setTimeout(function(){ callback($) })
```

Comment:
jQuery codebase is not our codebase. Explicit code is our goal rather than terseness. We use `0` in other parts of the library too.